### PR TITLE
New pages: HTMLMeterElement properties

### PR DESCRIPTION
**Pull request by estelle**
Part of #35544

### CODE REVIEW

File: files/en-us/web/api/htmlmeterelement/high/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+{{ APIRef("HTML DOM") }}
+
+The **`high`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the high boundary of the {{htmlelement("meter")}} element as a `double`. It reflects the element's [`high`](/en-US/docs/Web/HTML/Element/meter#high) attribute, or the value of `max` if not defined. The value of `high` is clamped by the `low` and `max` values.
+
+This property can also be set directly, for example to set a default value based on some condition.
+
+## Value
+
+A number. The value of `high`, but not less than {{DOMxRef("HTMLMeterElement.low")}} nor greater than {{DOMxRef("HTMLMeterElement.max")}}.
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlmeterelement/high/index.md:17]:
```suggestion
A number that is not less than {{DOMxRef("HTMLMeterElement.low")}} nor greater than {{DOMxRef("HTMLMeterElement.max")}}.
```

File: files/en-us/web/api/htmlmeterelement/low/index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+{{ APIRef("HTML DOM") }}
+
+The **`low`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the low boundary of the {{htmlelement("meter")}} element as a `double`. It reflects the element's [`low`](/en-US/docs/Web/HTML/Element/meter#low) attribute, or the value of `min` if not defined. The value of `low` is clamped by the `min` and `max` values.
+
+This property can also be set directly, for example to set a default value based on some condition.
+
+## Value
+
+A number. The value of `low`, but not less than {{DOMxRef("HTMLMeterElement.min")}} nor greater than {{DOMxRef("HTMLMeterElement.max")}}.
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlmeterelement/low/index.md:17]:
```suggestion
A number that is not less than {{DOMxRef("HTMLMeterElement.min")}} nor greater than {{DOMxRef("HTMLMeterElement.max")}}.
```

File: files/en-us/web/api/htmlmeterelement/high/index.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+title: "HTMLMeterElement: high property"
+short-title: high
+slug: Web/API/HTMLMeterElement/high
+page-type: web-api-instance-property
+browser-compat: api.HTMLMeterElement.high
+---
+
+{{ APIRef("HTML DOM") }}
+
+The **`high`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the high boundary of the {{htmlelement("meter")}} element as a `double`. It reflects the element's [`high`](/en-US/docs/Web/HTML/Element/meter#high) attribute, or the value of `max` if not defined. The value of `high` is clamped by the `low` and `max` values.
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlmeterelement/high/index.md:11]:
```suggestion
The **`high`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the high boundary of the {{htmlelement("meter")}} element as a floating-point number. It reflects the element's [`high`](/en-US/docs/Web/HTML/Element/meter#high) attribute, or the value of `max` if not defined. The value of `high` is clamped by the `low` and `max` values.
```

File: files/en-us/web/api/htmlmeterelement/low/index.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+title: "HTMLMeterElement: low property"
+short-title: low
+slug: Web/API/HTMLMeterElement/low
+page-type: web-api-instance-property
+browser-compat: api.HTMLMeterElement.low
+---
+
+{{ APIRef("HTML DOM") }}
+
+The **`low`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the low boundary of the {{htmlelement("meter")}} element as a `double`. It reflects the element's [`low`](/en-US/docs/Web/HTML/Element/meter#low) attribute, or the value of `min` if not defined. The value of `low` is clamped by the `min` and `max` values.
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlmeterelement/low/index.md:11]:
```suggestion
The **`low`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the low boundary of the {{htmlelement("meter")}} element as a floating-point number. It reflects the element's [`low`](/en-US/docs/Web/HTML/Element/meter#low) attribute, or the value of `min` if not defined. The value of `low` is clamped by the `min` and `max` values.
```

File: files/en-us/web/api/htmlmeterelement/max/index.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+title: "HTMLMeterElement: max property"
+short-title: max
+slug: Web/API/HTMLMeterElement/max
+page-type: web-api-instance-property
+browser-compat: api.HTMLMeterElement.max
+---
+
+{{ APIRef("HTML DOM") }}
+
+The **`max`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the maximum value of the {{htmlelement("meter")}} element as a `double`. It reflects the element's [`max`](/en-US/docs/Web/HTML/Element/meter#max) attribute, or the `min` value if no `max` is set, or `1` if neither the `min` or the `max` is defined.
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlmeterelement/max/index.md:11]:
```suggestion
The **`max`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the maximum value of the {{htmlelement("meter")}} element as a floating-point number. It reflects the element's [`max`](/en-US/docs/Web/HTML/Element/meter#max) attribute, or the `min` value if no `max` is set, or `1` if neither the `min` or the `max` is defined.
```

File: files/en-us/web/api/htmlmeterelement/max/index.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+This property can also be set directly, for example to set a default value based on some condition.
+
+## Value
+
+A number. The value of `max`, or the value of {{DOMxRef("HTMLMeterElement.min")}} or `1` if not defined.
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlmeterelement/max/index.md:17]:
```suggestion
A number.
```

File: files/en-us/web/api/htmlmeterelement/min/index.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+title: "HTMLMeterElement: min property"
+short-title: min
+slug: Web/API/HTMLMeterElement/min
+page-type: web-api-instance-property
+browser-compat: api.HTMLMeterElement.min
+---
+
+{{ APIRef("HTML DOM") }}
+
+The **`min`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the mimimum value of the {{htmlelement("meter")}} element as a number. It reflects the element's [`min`](/en-US/docs/Web/HTML/Element/meter#min) attribute, or `0` if no `min` is defined.
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlmeterelement/min/index.md:11]:
```suggestion
The **`min`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the mimimum value of the {{htmlelement("meter")}} element as a floating-point number. It reflects the element's [`min`](/en-US/docs/Web/HTML/Element/meter#min) attribute, or `0` if no `min` is defined.
```

File: files/en-us/web/api/htmlmeterelement/optimum/index.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+title: "HTMLMeterElement: optimum property"
+short-title: optimum
+slug: Web/API/HTMLMeterElement/optimum
+page-type: web-api-instance-property
+browser-compat: api.HTMLMeterElement.optimum
+---
+
+{{ APIRef("HTML DOM") }}
+
+The **`optimum`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the optimum boundary of the {{htmlelement("meter")}} element as a `double`. It reflects the element's [`optimum`](/en-US/docs/Web/HTML/Element/meter#optimum) attribute, or the midpoint between `min` and `max` values if not defined. The value of `optimum` is clamped by the `min` and `max` values.
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlmeterelement/optimum/index.md:11]:
```suggestion
The **`optimum`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the optimum boundary of the {{htmlelement("meter")}} element as a floating-point number. It reflects the element's [`optimum`](/en-US/docs/Web/HTML/Element/meter#optimum) attribute, or the midpoint between `min` and `max` values if not defined. The value of `optimum` is clamped by the `min` and `max` values.
```

File: files/en-us/web/api/htmlmeterelement/optimum/index.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+This property can also be set directly, for example to set a default value based on some condition.
+
+## Value
+
+A number. The value of `optimum`, or the midpoint between {{DOMxRef("HTMLMeterElement.min")}} and {{DOMxRef("HTMLMeterElement.max")}} if not defined.
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlmeterelement/optimum/index.md:17]:
```suggestion
A number. Defaults to the midpoint between {{DOMxRef("HTMLMeterElement.min")}} and {{DOMxRef("HTMLMeterElement.max")}} if not defined.
```

File: files/en-us/web/api/htmlmeterelement/value/index.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+title: "HTMLMeterElement: value property"
+short-title: value
+slug: Web/API/HTMLMeterElement/value
+page-type: web-api-instance-property
+browser-compat: api.HTMLMeterElement.value
+---
+
+{{ APIRef("HTML DOM") }}
+
+The **`value`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the current value of the {{htmlelement("meter")}} element as a `double`. It reflects the element's [`value`](/en-US/docs/Web/HTML/Element/meter#value) attribute. If no `value` is set, the {{DOMxRef("HTMLMeterElement.min")}} value or `0`, whichever is greater.
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlmeterelement/value/index.md:11]:
```suggestion
The **`value`** property of the {{DOMxRef("HTMLMeterElement")}} interface represents the current value of the {{htmlelement("meter")}} element as a floating-point number. It reflects the element's [`value`](/en-US/docs/Web/HTML/Element/meter#value) attribute. If no `value` is set, it is the {{DOMxRef("HTMLMeterElement.min")}} value or `0`, whichever is greater.
```

File: files/en-us/web/api/htmlmeterelement/value/index.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+
+This property can also be set directly, for example to set a default value based on some condition.
+
+## Value
+
+A number. The value of `value`, or either {{DOMxRef("HTMLMeterElement.min")}} or `0`, whichever if greater, if not defined.
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlmeterelement/value/index.md:17]:
```suggestion
A number. Defaults to either {{DOMxRef("HTMLMeterElement.min")}} or `0`, whichever if greater, if not defined.
```

**Review by Josh-Cena — COMMENTED** [review]
Almost good

**Comment by estelle** [comment]
Thanks @Josh-Cena

**Review by Josh-Cena — APPROVED** [review]
Looks good, thanks

**Pull request merged**